src/cart/crt_hg_proc.c

```diff
@@ -671,19 +674,25 @@ crt_proc_out_common(crt_proc_t proc, crt_rpc_output_t *data)
 			struct crt_common_hdr *hdr = &rpc_priv->crp_reply_hdr;
 
 			if (crt_is_service()) {
+				uint64_t clock_drift;
+
 				rc = crt_hlc_get_msg(hdr->cch_hlc,
-						     NULL /* hlc_out */);
+						     NULL /* hlc_out */,
+						     &clock_drift);
 				if (rc != 0) {
 					REPORT_HLC_SYNC_ERR("failed to sync "
 							    "HLC for reply: "
 							    "opc=%x ts="DF_U64
+							    " drift="DF_U64
```

Comment:
minor nit...the macro could potentially take opc, ts, drift, and from as arguments

Reply:
I didn't want to change existing macro in case LiWei had other plans for it later where maybe different stuff will be printed that doesnt contain opc/ts/drift etc..